islandora_transforms/BPL_MODS_to_solr.xslt

```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- BPL requires some indexing of cases not handled by standard MODS slurping -->
+<xsl:stylesheet version="1.0"
+  xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+  xmlns:foxml="info:fedora/fedora-system:def/foxml#"
+  xmlns:mods="http://www.loc.gov/mods/v3"
+  exclude-result-prefixes="mods">
+
+  <!--Non-date-typed regular name/namePart -->
+  <xsl:template mode="slurp_for_bpl" match="mods:mods/mods:name[@type='personal']/mods:namePart[not(@type='date')]">
+    <xsl:call-template name="write_bpl_field">
+      <xsl:with-param name="field_name" select="'name_personal_namePart_not_date'"/>
+      <xsl:with-param name="content" select="normalize-space()"/>
+    </xsl:call-template>
+  </xsl:template>
+
+  <!-- Non-date-typed subject name/namePart -->
+  <xsl:template mode="slurp_for_bpl" match="mods:mods/mods:subject/mods:name[@type='personal']/mods:namePart[not(@type='date')]">
```

Comment:
Won't get executed because 

```xml
<!-- Concatenate geographic elements together using double-dashes -->
<xsl:template mode="slurp_for_bpl" match="mods:mods/mods:subject">
``` 

Matches first and it doesn't call 

```xml
<xsl:apply-templates mode="slurp_for_bpl" match="mods:name">
``` 

So the XSLT doesn't recurse any further.

Also this template also matches the next template.

```xml
<!-- Make sure subject/name/nameParts of all types are being indexed (no
        non-typed field is currently indexed for subject/name) -->
<xsl:template mode="slurp_for_bpl" match="mods:mods/mods:subject/mods:name/mods:namePart">
```

So they one or the other will run but not both.